### PR TITLE
rename import of css file

### PR DESCRIPTION
**Pull request by shadowfree**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,6 +3,6 @@ import ReactDOM from "react-dom";
 
 import App from "./App";
 
-import "./index.css";
+import "./app.css";
```
Comment (RudyWeber) [inline src/index.js]: As I said in the previous PR, there is no reason to import `app.css` in this file.
`app.css` is meant to be imported in `App.js` and nowhere else.

The fix should be to simply delete this useless import. `index.css` doesn't even exist.

Comment (shadowfree) [reply src/index.js]: ah ok, sorry ! I remove it

**Comment by RudyWeber** [comment]
The only thing to do in this PR is to remove `import "index.css";` from the `index.js` file. Nothing else.
`app.css` is needed in `App.js` and should not be removed nor moved into `index.js`.

File: src/App.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import HangManDrawing from "./components/HangManDrawing";
 
 import { getWord, lettersToUnderscores, revealLetter } from "./services/words";
 
-import "./app.css";
+import './app.css'
```
Comment (RudyWeber) [inline src/App.js]: Coding style.

You simply should discard your changes of `src/App.js`.

File: src/App.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import HangManDrawing from "./components/HangManDrawing";
 
 import { getWord, lettersToUnderscores, revealLetter } from "./services/words";
 
-import "./app.css";
+import "./app.css"
```
Comment (RudyWeber) [inline src/App.js]: `;` missing.
Again, as said before, you should discard your changes of `src/App.js`. There shouldn't be any changes in this file for this PR.

**Pull request merged**